CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Fix: [#982] Incorrect rating calculation for cargo causing penalty for fast vehicles.
 - Fix: [#1008] Inability to decrease max altitude for trees in landscape editor.
 - Fix: [#1016] Incorrect detection of station causing incorrect smoke sounds.
+- Fix: [#1022] Fix access to embedded object name
```

Comment:
This isn't very helpful to the average changelog reader. What user-facing problem is it solving? e.g.
```suggestion
- Fix: [#1022] Crash due to invalid access to embedded object name.
```

Reply:
Updated